index.html

```diff
@@ -3,17 +3,108 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <title>Karma</title>
     <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" rel='stylesheet'  href="media.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <title>Karma</title>
 </head>
+
 <body>
+    <div class="navbar">
+        <div class="logo">
+            <img src="img/karma-logo.svg" alt="logo">
+        </div>
+
+
+        <div class="hamburger-menu">
+            <input id="menu__toggle" type="checkbox" />
+            <label class="menu__btn" for="menu__toggle">
+              <span></span>
+            </label>
+        
+            <ul class="menu__box">
+              <li><a class="menu__item" href="Meet Karma">Meet Karma</a></li>
```

Comment:
```suggestion
              <li><a class="menu__item" href="Meet Karma"><strong>Meet Karma</strong></a></li>
```

Reply:
I added you by mistake the second time. Thanks anyway! I would probably choose bold instead of strong. I am not sure what strong does. I will test it.